tests/test_integration.py

```diff
@@ -31,7 +31,7 @@
 # XXX: Make it hashable to silence warnings; see if this can be done upstream
 # This isn't a huge deal, but our testing context mangers for asserting
 # exceptions need hashability
-TOmniSciException.__hash__ = lambda x: id(x)
+TDBException.__hash__ = lambda x: id(x)
```

Comment:
Nit:
```suggestion
TDBException.__hash__ = id
```

Reply:
Yeah I tried to not touch anything else than rebranding, but if wanted I can do a general cleaning on the code as well.